packages/block-template-react/src/dev.tsx

```diff
@@ -17,6 +18,7 @@ const DevApp = () => {
         entityId: "test-block-1",
         properties: { name: "World" },
       }}
+      blockInfo={packageJSON.blockprotocol}
```

Comment:
This updates `block-template-react` to pull blockInfo from the block's `package.json#protocol` field.